### PR TITLE
zh-CN: update the translation of Games/Anatomy

### PR DESCRIPTION
**Pull request by zjffun**
Details: https://zjffun.github.io/translated-content/gh-pages/7cafbadea71a7b796ceb398c4a8832625a1997e2.html

And fix a punctuation: https://github.com/mdn/translated-content/compare/70d01afb6c720fa83828e0ee75268c42cc2a9239..9ab59e6b611ea65895fe10f850c852f7b519ee40

And fix ref of Date https://github.com/mdn/translated-content/compare/9ab59e6b611ea65895fe10f850c852f7b519ee40..5141e476c753eed5817fb153214a92551ff03906

### CODE REVIEW

File: files/zh-cn/games/anatomy/index.md [hunk 1]
```diff
@@ -46,121 +47,133 @@ main(); //开始循环
...
 1. 启动一个新帧（而之前的帧由显示处理）。
 2. 遍历 requestAnimationFrame 回调并调用它们。
 3. 当上面的回调停止控制主线程时，执行垃圾收集和其他帧任务。
-4. 睡眠（除非事件打断了浏览器的小睡），直到显示器准备好你的图像（VSYNC）并重复。
+4. 睡眠（除非事件打断了浏览器的小睡），直到显示器准备好你的图像（[VSync](https://www.techopedia.com/definition/92/vertical-sync-vsync)）并重复。
 
 您可以考虑开发实时应用程序，因为有时间做工作。所有上述步骤都必须在每 16 毫秒内进行一次，以保持 60 赫兹的显示效果。浏览器会尽可能早地调用您的代码，从而给它最大的计算时间。您的主线程通常会启动一些甚至不在主线程上的工作负载（如 WebGL 的中的光栅化或着色器）。在浏览器使用其主线程管理垃圾收集，其他任务或处理异步事件时，可以在 Web Worker 或 GPU 上执行长时间的计算。
 
-当我们讨论预算时，许多网络浏览器都有一个称为高分辨率时间的工具.{{ domxref("Date") }} 对象不再是计时事件的识别方法，因为它非常不精确，可以由系统时钟进行修改。另一方面，高分辨率的时间计算自 navigationStart（当上一个文档被卸载时）的毫秒数。这个值以小数的精度返回，精确到千分之一毫秒。它被称为{{ domxref("DOMHighResTimeStamp") }}，但是，无论出于什么目的和目的，都认为它是一个浮点数。
+当我们讨论预算时，许多网络浏览器都有一个称为高分辨率时间的工具。{{ domxref("Date") }} 对象不再是计时事件的识别方法，因为它非常不精确，可以由系统时钟进行修改。另一方面，高分辨率的时间计算自`navigationStart`（当上一个文档被卸载时）的毫秒数。这个值以小数的精度返回，精确到千分之一毫秒。它被称为{{ domxref("DOMHighResTimeStamp") }}，但是，无论出于什么目的和目的，都认为它是一个浮点数。
```
Comment (zjffun) [inline files/zh-cn/games/anatomy/index.md:124]: Need to update if https://github.com/mdn/content/pull/20566 merged.

File: files/zh-cn/games/anatomy/index.md [hunk 2]
```diff
@@ -24,143 +25,155 @@ slug: Games/Anatomy
...
 
-现在，我们在 MyGame 名称空间中声明了一个变量`stopMain`，其值为主循环最后调用`requestAnimationFrame()`时返回的 ID。任何时候，我们可以通过告诉浏览器取消与 ID 相对应的请求来停止主循环。
+现在，我们在 MyGame 名称空间中声明了一个变量`stopMain`，其值为主循环最后调用 `requestAnimationFrame()` 时返回的 ID。任何时候，我们可以通过告诉浏览器取消与 ID 相对应的请求来停止主循环。
 
 ```js
 window.cancelAnimationFrame(MyGame.stopMain);
 ```
 
-在 JavaScript 的中编写**主循环**的关键在于，考虑到任何会驱动你行为的事件，并注意不同的系统是如何相互作用的。您可能拥有多个由多个不同类型的事件驱动的组件。这看起来像是不必要的复杂性，但它可能就是一个很好的优化（当然，不一定是这样的）。问题是，您并不是在编写一个典型的主循环。在 Java 脚本中，您使用的是浏览器的主循环，并且您正在尝试这样做。
+
```
Comment (yin1999) [inline files/zh-cn/games/anatomy/index.md:108]:
```suggestion
```

File: files/zh-cn/games/anatomy/index.md [hunk 3]
```diff
@@ -24,143 +25,155 @@ slug: Games/Anatomy
...
+在 JavaScript 的中编写主循环的关键在于，考虑到任何会驱动你行为的事件，并注意不同的系统是如何相互作用的。您可能拥有多个由多个不同类型的事件驱动的组件。这看起来像是不必要的复杂性，但它可能就是一个很好的优化（当然，不一定是这样的）。问题是，您并不是在编写一个典型的主循环。在 JavaScript 中，您使用的是浏览器的主循环，并且您正在尝试这样做。
 
 ## 用 JavaScript 构建一个更优化的主循环
 
-基本上，在 JavaScript 的中，浏览器有它自己的主循环，而你的代码存在于循环某些阶段。上面描述的主循环，试图避免脱离浏览器的控制。这种主循环附着于`window.requestAnimationFrame()`方法，该方法将在浏览器的下一帧中执行，具体取决于浏览器如何与将其自己的主循环关联起来。[W3C 规范](/zh-CN/docs/Games/Anatomy)并没有真正定义什么时候浏览器必须执行 requestAnimationFrame 回调。这有一个好处，浏览器厂商可以自由地实现他们认为最好的解决方案，并随着时间的推移进行调整。
+基本上，在 JavaScript 的中，浏览器有它自己的主循环，而你的代码存在于循环某些阶段。上面描述的主循环，试图避免脱离浏览器的控制。这种主循环附着于 `window.requestAnimationFrame()` 方法，该方法将在浏览器的下一帧中执行，具体取决于浏览器如何与将其自己的主循环关联起来。[W3C 规范中的 requestAnimationFrame](https://www.w3.org/TR/animation-timing/) 并没有真正定义什么时候浏览器必须执行 requestAnimationFrame 回调。这有一个好处，浏览器厂商可以自由地实现他们认为最好的解决方案，并随着时间的推移进行调整。
```
Comment (yin1999) [inline files/zh-cn/games/anatomy/index.md:113]:
```suggestion
基本上，在 JavaScript 的中，浏览器有它自己的主循环，而你的代码存在于循环某些阶段。上面描述的主循环，试图避免脱离浏览器的控制。这种主循环附着于 `window.requestAnimationFrame()` 方法，该方法将在浏览器的下一帧中执行，具体取决于浏览器如何与将其自己的主循环关联起来。[W3C 的 requestAnimationFrame 规范](https://www.w3.org/TR/animation-timing/)并没有真正定义什么时候浏览器必须执行 requestAnimationFrame 回调。这有一个好处，浏览器厂商可以自由地实现他们认为最好的解决方案，并随着时间的推移进行调整。
```

File: files/zh-cn/games/anatomy/index.md [hunk 4]
```diff
@@ -24,143 +25,155 @@ slug: Games/Anatomy
...
 
 现代版的 Firefox 和 Google Chrome（可能还有其他版本）试图在框架的时间片的开始时将请求 AnimationFrame 回调与它们的主线程进行连接。因此，浏览器的主线程看起来就像下面这样：
```
Comment (yin1999) [inline files/zh-cn/games/anatomy/index.md:115]:
```suggestion
现代版的 Firefox 和 Google Chrome（可能还有其他版本）试图在框架的时间片的开始时尝试将 `requestAnimationFrame` 回调与它们的主线程进行连接。因此，浏览器的主线程看起来就像下面这样：
```

File: files/zh-cn/games/anatomy/index.md [hunk 5]
```diff
@@ -24,143 +25,155 @@ slug: Games/Anatomy
...
 
 1. 启动一个新帧（而之前的帧由显示处理）。
 2. 遍历 requestAnimationFrame 回调并调用它们。
```
Comment (yin1999) [inline files/zh-cn/games/anatomy/index.md:118]:
```suggestion
2. 遍历 `requestAnimationFrame` 回调并调用它们。
```

File: files/zh-cn/games/anatomy/index.md [hunk 6]
```diff
@@ -24,143 +25,155 @@ slug: Games/Anatomy
...
-4. 睡眠（除非事件打断了浏览器的小睡），直到显示器准备好你的图像（VSYNC）并重复。
+4. 睡眠（除非事件打断了浏览器的小睡），直到显示器准备好你的图像（[VSync](https://www.techopedia.com/definition/92/vertical-sync-vsync)）并重复。
 
 您可以考虑开发实时应用程序，因为有时间做工作。所有上述步骤都必须在每 16 毫秒内进行一次，以保持 60 赫兹的显示效果。浏览器会尽可能早地调用您的代码，从而给它最大的计算时间。您的主线程通常会启动一些甚至不在主线程上的工作负载（如 WebGL 的中的光栅化或着色器）。在浏览器使用其主线程管理垃圾收集，其他任务或处理异步事件时，可以在 Web Worker 或 GPU 上执行长时间的计算。
 
-当我们讨论预算时，许多网络浏览器都有一个称为高分辨率时间的工具.{{ domxref("Date") }} 对象不再是计时事件的识别方法，因为它非常不精确，可以由系统时钟进行修改。另一方面，高分辨率的时间计算自 navigationStart（当上一个文档被卸载时）的毫秒数。这个值以小数的精度返回，精确到千分之一毫秒。它被称为{{ domxref("DOMHighResTimeStamp") }}，但是，无论出于什么目的和目的，都认为它是一个浮点数。
+当我们讨论预算时，许多网络浏览器都有一个称为高分辨率时间的工具。{{jsxref("Date")}} 对象不再是计时事件的识别方法，因为它非常不精确，可以由系统时钟进行修改。另一方面，高分辨率的时间计算自 `navigationStart`（当上一个文档被卸载时）的毫秒数。这个值以小数的精度返回，精确到千分之一毫秒。它被称为{{ domxref("DOMHighResTimeStamp") }}，但是，无论出于什么目的和目的，都认为它是一个浮点数。
 
-注意：系统（硬件或软件）不能达到微秒精度，可以提供毫秒精度的最小值然而，如果他们能够做到这一点，他们就应该提供 0.001 的准确性。
+> **备注：** 系统（硬件或软件）不能达到微秒精度，可以提供毫秒精度的最小值然而，如果他们能够做到这一点，他们就应该提供 0.001 的准确性。
```
Comment (yin1999) [inline files/zh-cn/games/anatomy/index.md:126]:
```suggestion
> **备注：** 系统（硬件或软件）不能达到微秒精度，可以提供毫秒精度的最小值然而，如果他们能够做到这一点，他们就应该提供 0.001 毫秒的准确性。
```

File: files/zh-cn/games/anatomy/index.md [hunk 7]
```diff
@@ -192,17 +206,17 @@ var tNow = window.performance.now();
 
 一种常见的技术是以恒定的频率更新模拟，然后绘制尽可能多的（或尽可能少的）实际帧。更新方法可以继续循环，而不用考虑用户看到的内容。绘图方法可以查看最后的更新以及发生的时间。由于绘制知道何时表示，以及上次更新的模拟时间，它可以预测为用户绘制一个合理的框架。这是否比官方更新循环更频繁（甚至更不频繁）无关紧要。更新方法设置检查点，并且像系统允许的那样频繁地，渲染方法画出周围的时间。在 Web 标准中分离更新方法有很多种方法：
 
-- 绘制`requestAnimationFrame`并更新 {{ domxref("window.setInterval") }}或{{domxref("window.setTimeout")}}。
+- 绘制 `requestAnimationFrame` 并更新 {{domxref("setInterval()")}} 或 {{domxref("setTimeout()")}}。
 
   - 即使在未聚焦或最小化的情况下，使用处理器时间，也可能是主线程，并且可能是传统游戏循环的工件（但是很简单）。
 
-- 绘制`requestAnimationFrame`和更新一个`setInterval`或`setTimeout`一个[Web 工作者](/zh-CN/docs/Web/Guide/Performance/Using_web_workers)。
+- 绘制 `requestAnimationFrame` 和更新一个 `setInterval` 或 `setTimeout` 一个 [Web Worker](/zh-CN/docs/Web/API/Web_Workers_API/Using_web_workers)。
```
Comment (yin1999) [inline files/zh-cn/games/anatomy/index.md:213]:
```suggestion
- 绘制 `requestAnimationFrame` 并在 [Web Worker](/zh-CN/docs/Web/API/Web_Workers_API/Using_web_workers) 的 `setInterval` 或 `setTimeout` 中对其进行更新。
```

File: files/zh-cn/games/anatomy/index.md [hunk 8]
```diff
@@ -266,31 +280,32 @@ _注意：这个例子，具体来说，需要进行技术审查。_
...
-  main(performance.now()); //开始循环
+  main(performance.now()); // 开始循环
 })();
 ```
+<!-- prettier-ignore-end -->
 
-另一个选择是简单地做一些事情不那么频繁。如果您的更新循环的一部分难以计算但对时间不敏感，则可以考虑缩小其频率，理想情况下，在延长的时间段内将其扩展成块。这是一个隐含的例子，在火炮博物馆的炮兵游戏中，他们[调整垃圾发生率](http://blog.artillery.com/2012/10/browser-garbage-collection-and-framerate.html)来优化垃圾收集。显然，清理资源不是时间敏感的（特别是如果整理比垃圾本身更具破坏性）。
+另一个选择是简单地做一些事情不那么频繁。如果您的更新循环的一部分难以计算但对时间不敏感，则可以考虑缩小其频率，理想情况下，在延长的时间段内将其扩展成块。这是一个隐含的例子，在火炮博物馆的炮兵游戏中，他们[调整垃圾发生率](https://web.archive.org/web/20161021030645/http://blog.artillery.com/2012/10/browser-garbage-collection-and-framerate.html)来优化垃圾收集。显然，清理资源不是时间敏感的（特别是如果整理比垃圾本身更具破坏性）。
 
 这也可能适用于您自己的一些任务。那些是当可用资源成为关注点时的好候选人。
```
Comment (yin1999) [inline files/zh-cn/games/anatomy/index.md:302]:
```suggestion
这也可能适用于你自己的一些任务。那些是当可用资源成为关注点时的好候选人。
```

**Review by yin1999 — COMMENTED** [review]
Thanks a lot. I've found some translation issue.

**Review by yin1999 — APPROVED** [review]
Thanks @zjffun 👍

**Pull request merged**